app/Models/TrainCheckin.php

```diff
@@ -97,7 +97,7 @@ public function getDestinationStopoverAttribute(): TrainStopover {
                                                ->first();
         if ($stopOver == null) {
             //To support legacy data, where we don't save the stopovers in the stopovers table, yet.
-            Log::error('TrainCheckin #' . $this->id . ': Destination stopover not found. Created a new one.');
+            Log::info('TrainCheckin #' . $this->id . ': Destination stopover not found. Created a new one.');
```

Comment:
This is also an error, as this should never happen...

Reply:
See above. It will always.